android/app/src/androidTest/java/com/novoda/bonfire/user/UsersActivityTest.java

```diff
@@ -127,7 +127,7 @@ private void channelOwnersUpdateAfterEveryChange() {
         doAnswer(new Answer<Observable<DatabaseResult<Users>>>() {
             @Override
             public Observable<DatabaseResult<Users>> answer(InvocationOnMock invocation) throws Throwable {
-                return subject.asObservable().observeOn(AndroidSchedulers.mainThread());
+                return subject.observeOn(AndroidSchedulers.mainThread());
```

Comment:
`observeOn` returns an `Observable` straight away, what is the advantage of using `hide`?